css/_uswds-theme-custom-styles.scss

```diff
@@ -646,6 +650,28 @@ body {
 
 // Footer --------------- //
 
+.touchpoints__container {
+  padding-left: units(2);
+  padding-right: units(2);
+  @include at-media("desktop") {
+    display: flex;
+    justify-content: flex-end;
+  }
+  @include at-media("widescreen") {
+    bottom: 0;
+    position: sticky;
+    z-index: z-index("top");
+  }
```

Comment:
Why not make it sticky by default?

Reply:
It felt intrusive to me to have it cover content in small browser windows. I could switch it to stick at something like tablet-lg instead of widescreen though. What do you think?

Reply:
Will leave as is for now - we can move it in the future if needed.